### ISSUE TITLE
Enhanced UX for Downloaded videos

### ISSUE THREAD

**Issue by IkelAtomig** [libre-tube/LibreTube#2527]
### Describe your suggested feature

Currently all the download videos play in full screen mode. I think this is an limitation of Exoplayer.

What I suggest is replicate it similar to YouTube. Play the video in the top 1/4 of the screen and below its show other downloaded videos or play them as a series of videos in playlist.

The normal Video watching experience without hassle.

And for videos where only audio is downloaded. Instead of showing a Black screen, better to show the thumbnail instead.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Aqwuty** [comment]
You can play downloaded videos in portrait mode

**Comment by IkelAtomig** [comment]
> You can play downloaded videos in portrait mode

Not within the LibreTube App. I have tried. It's locked in Landscape mode.

**Comment by Aqwuty** [comment]
I can play.🤔

**Comment by IkelAtomig** [comment]
> I can play the video.

I could do but in Potrait mode ?

**Comment by Bnyro** [comment]
An audio only player for downloads is on the road with #3525.
I don't see the point of modifying the current offline video player, thus the issues will be closed once the linked PR is merged.
